src/node.cc

```diff
@@ -4188,6 +4188,11 @@ void Init(int* argc,
   V8::SetFlagsFromString(NODE_V8_OPTIONS, sizeof(NODE_V8_OPTIONS) - 1);
 #endif
 
+  // Allow for environment set preserving symlinks.
+  if (auto preserve_symlinks = secure_getenv("NODE_PRESERVE_SYMLINKS")) {
+     config_preserve_symlinks = (*preserve_symlinks != '\0');
```

Comment:
For consistency with other env variables, it should be enabled when set to `1` and disabled otherwise.